test/e2e/metrics_util.go

```diff
@@ -40,7 +40,9 @@ import (
 
 const (
 	podStartupThreshold           time.Duration = 5 * time.Second
-	listPodLatencyThreshold       time.Duration = 2 * time.Second
+	listPodLatencySmallThreshold  time.Duration = 2 * time.Second
+	listPodLatencyMediumThreshold time.Duration = 4 * time.Second
+	listPodLatencyLargeThreshold  time.Duration = 15 * time.Second
```

Comment:
We should down all of them to 1s in the fullness of time (not to 8) :P 

Can you please add a TODO to decrease those limits?